src/commands/domains/help.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
I guess you are waiting until the command names are finalized before populating the help?